includes/Embed_Base.php

```diff
@@ -55,6 +62,21 @@ public function init() {
 		wp_register_script( self::STORY_PLAYER_HANDLE, 'https://cdn.ampproject.org/amp-story-player-v0.js', [], 'v0', false );
 		wp_register_style( self::STORY_PLAYER_HANDLE, 'https://cdn.ampproject.org/amp-story-player-v0.css', [], 'v0' );
 
+		// Registering a style without a `src` allows us to just use the inline style below
+		// without needing an external stylesheet.
+		wp_register_style(
+			self::SCRIPT_HANDLE,
+			'',
+			[],
+			WEBSTORIES_VERSION
+		);
+
+		$css = file_get_contents( WEBSTORIES_PLUGIN_DIR_PATH . 'includes/assets/embed.css' );
+
+		if ( $css ) {
+			wp_add_inline_style( self::SCRIPT_HANDLE, $css );
+		}
```

Comment:
Why the `if ( $css )`? Is that to handle the case where `embed.css` is not built? If so, wouldn't it be better to use a `file_exists()` check around this entire block?

Reply:
There’s no build step, but `file_get_contents` can return false

Reply:
But it will also issue a PHP warning at the same time.

Reply:
`file_exists` sounds reasonable, but without the `$css` check PHPStan will still rightfully say "wp_add_inline_style expects string, string|false given."

Reply:
Got it. Could also to `is_string( $css )` I suppose.